src/js/components/Marathon.jsx

```diff
@@ -44,7 +48,11 @@ var Marathon = React.createClass({
 
     this.bindKeyboardShortcuts();
 
-    this.startPolling();
+    PluginStore.once(PluginEvents.BOOTSTRAP_COMPLETE, () => {
+      this.startPolling();
+    });
+
+    PluginStore.bootstrap();
```

Comment:
Please respect the Flux design pattern and use an action to start the bootstrapping. 

_Note: Refactored in #609._